src/plugins/intel_gpu/src/plugin/transformations/convert_fc_to_compressed.cpp

```diff
@@ -42,7 +42,10 @@ ConvertFullyConnectedToFullyConnectedCompressed::ConvertFullyConnectedToFullyCon
     auto convert_m = wrap_type<ov::op::v0::Convert>({weights_m});
 
     auto sub_const_m = wrap_type<ov::op::v0::Constant>(consumers_count(1));
-    auto subtract_m = wrap_type<ov::op::v1::Subtract>({convert_m, sub_const_m});
+    auto sub_convert_m = wrap_type<ov::op::v0::Convert>({sub_const_m});
```

Comment:
I didn't get why this is needed. Subtract constant is supposed to be folded prior to this pass (that's why `MarkDequantizationSubgraph` is called with `fold_subtract_const=true`). So to me it seems that the original problem is not related to this pass and should be fixed in a different way